src/frontend/next/src/components/SearchPage/SearchResults.tsx

```diff
@@ -0,0 +1,84 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { useSWRInfinite } from 'swr';
+import { Container, Box, Grid, Typography, ListSubheader } from '@material-ui/core';
+import useSiteMetadata from '../../hooks/use-site-metadata';
+import Timeline from '../Timeline';
```

Comment:
Timeline is in now, can we get this rebased so I can test?

Reply:
Rebased!